system_suitesparse/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(system_suitesparse)
+
+find_package(catkin REQUIRED)
+
+list(APPEND CMAKE_MODULE_PATH "${system_suitesparse_SOURCE_DIR}/cmake")
+
+find_package(SUITESPARSE REQUIRED)
+
+install(DIRECTORY ${SUITESPARSE_INCLUDE_DIRS}
+        DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
```

Comment:
I don't think we need that. Instead, here, I would check whether suite-sparse was actually found (probing `SUITESPARSE_FOUND`, which should be set by the Find script) and if not fail with a `message(FATAL ..`.